SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialDispatcher.cpp

```diff
@@ -41,28 +53,73 @@ void USpatialDispatcher::ProcessOps(Worker_OpList* OpList)
 
 		// Components
 		case WORKER_OP_TYPE_ADD_COMPONENT:
-			StaticComponentView->OnAddComponent(Op->add_component);
-			Receiver->OnAddComponent(Op->add_component);
+			if (UOpCallbackTemplate** UserCallbackWrapper = UserOpCallbacks.Find(Op->add_component.data.component_id))
```

Comment:
Discussed offline. I think these ifs can be condensed by making two different dispatchers.